lib/index.js

```diff
@@ -10,10 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* globals process, console, Q, JSON, escape, verifyToken, crypto */
+/* globals process, console, SDK_VERSION */
 /*jshint maxcomplexity:15 */
 
-var SDK_VERSION = '<%= sdkversion %>'; // Will be replaced by grunt
+require('./vendor/polyfills');
```

Comment:
For base64, can you add it as a dependency from [npm](https://www.npmjs.com/package/Base64)? And agree, let's add a comment before both requires (one for base64, one for our vendor polyfills)